doc/source/rayserve/overview.rst

```diff
@@ -120,36 +122,43 @@ Note that this will not delete any associated backends, which can be reused for
 Backends
 ~~~~~~~~
 
-Backends are the logical structures for your business logic or models and 
+Backends are the logical structures for your business logic or models and
 how you specify what should happen when an endpoint is queried.
-To define a backend, first you must define the "handler" or the business logic you'd like to respond with. 
+To define a backend, first you must define the "handler" or the business logic you'd like to respond with.
 The input to this request will be a `Flask Request object <https://flask.palletsprojects.com/en/1.1.x/api/?highlight=request#flask.Request>`_.
-Once you define the function (or class) that will handle a request. 
+Once you define the function (or class) that will handle a request.
 You'd use a function when your response is stateless and a class when you
-might need to maintain some state (like a model). 
-For both functions and classes (that take as input Flask Requests), you'll need to 
+might need to maintain some state (like a model).
+For both functions and classes (that take as input Flask Requests), you'll need to
 define them as backends to RayServe.
 
-It's important to note that RayServe places these backends in individual workers, which are replicas of the model.
+It's important to note that RayServe places these backends in individual workers, which are replicas of the model. This means that any global state you need shared across workers must be in a Ray actor (or external data store), which is referenced from the function or regular class you specify for the backend.
+
+Here is an example that uses a function as a request handler:
 
 .. code-block:: python
-  
+
   def handle_request(flask_request):
     return "hello world"
 
+  serve.create_backend("simple_backend", handle_request)
+
+Here is a similar example that uses a class as a request handler:
+
+.. code-block:: python
+
   class RequestHandler:
     def __init__(self):
         self.msg = "hello, world!"
 
     def __call__(self, flask_request):
         return self.msg
 
-  serve.create_backend("simple_backend", handle_request)
-  serve.create_backend("simple_backend_class", RequestHandler)
+  serve.create_backend("simple_backend", RequestHandler)
 
-Lastly, we need to link the particular backend to the server endpoint. 
+Lastly, we need to link the particular backend to the server endpoint.
 To do that we'll use the ``link`` capability.
-A link is essentially a load-balancer and allow you to define queuing policies 
+A link is essentially a load-balancer and allow you to define queuing policies
```

Comment:
`link` doesn't exist anymore. While you're at it, could you update these lines to say `set_traffic`?